packages/x-codemod/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.0.5",
+  "version": "6.0.6",
```

Comment:
Up until now, we've synced this package release version with the other packages: https://www.npmjs.com/package/@mui/x-codemod?activeTab=versions
Do you think we should keep doing it? 🤔

Reply:
Yes, we did that, but I believe we were discussing that it was sort of anti-pattern that we'd like to avoid in the future.
Am I remembering incorrectly @flaviendelangle @alexfauquette?